subnet/subnet.go

```diff
@@ -96,5 +105,7 @@ type Manager interface {
 	GetNetworkConfig(ctx context.Context, network string) (*Config, error)
 	AcquireLease(ctx context.Context, network string, attrs *LeaseAttrs) (*Lease, error)
 	RenewLease(ctx context.Context, network string, lease *Lease) error
-	WatchLeases(ctx context.Context, network string, cursor interface{}) (WatchResult, error)
+	WatchLeases(ctx context.Context, network string, cursor interface{}) (LeaseWatchResult, error)
+	GetNetworks(ctx context.Context) ([]string, error)
```

Comment:
Done in the latest push.  I opted for "WatchNetworks" instead of GetNetworks, but could go either way if you really prefer GetNetworks.